pom.xml

```diff
@@ -615,6 +621,14 @@
                 <!-- Pass system props because the plugin doesn't take ENVs (and the config library doesn't care) -->
                 <jvmArg>-DHTTP_PORT=${http.test.port}</jvmArg>
               </jvmArgs>
+              <runArgs>
+                <runArg>-Dlibcal.client1.id="${libcal.client1.id}"</runArg>
+                <runArg>-Dlibcal.client1.secret="${libcal.client1.secret}"</runArg>
+                <runArg>-Dlibcal.client2.id="${libcal.client2.id}"</runArg>
+                <runArg>-Dlibcal.client2.secret="${libcal.client2.secret}"</runArg>
+                <runArg>-Dlibcal.token.endpoint="${libcal.token.endpoint}"</runArg>
+                <runArg>-Dlibcal.base.url="${libcal.base.url}"</runArg>
+              </runArgs>
```

Comment:
Why this change?

Reply:
If one wants to run the app using the Maven Vert.x plugin in what we've called "live" mode in the past. I used that a bit when trying to shortcut the long build cycle of the Docker container.